### PR TITLE
UefiPayloadPkg: Enhance build script for BuildEntryOnly

### PR DESCRIPTION
**Pull request by gguo11837463**

### CODE REVIEW

File: UefiPayloadPkg/MkFitImage.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+def BuildTianoImageNode(Fdt, InfoHeader, ParentNode):
+    Prestr = InfoHeader.LoadAddr
+    if Prestr[:2] != '0x':
+        print('Error hex format!')
+        return
+    Hexint = int(Prestr[2:], 16)
+    libfdt.fdt_setprop_u32(Fdt, ParentNode, 'load', Hexint)
+
+    Prestr = InfoHeader.LoadAddr
```
Comment (jameslu8) [inline UefiPayloadPkg/MkFitImage.py:90]: RelocStart

Comment (jameslu8) [inline UefiPayloadPkg/MkFitImage.py:90]: LoadAddr -> RelocStart

File: UefiPayloadPkg/MkFitImage.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+from elftools.elf.elffile import ELFFile
+
+FDT_DEFAULT_SIZE = 1024
+FDT_ALIGN        = 16
+
+class FIT_IMAGE_INFO_HEADER:
+    _pack_ = 1
+    _fields_ = [
+        ('Compatible',    str),
+        ('UpVersion',     int),
```
Comment (jameslu8) [inline UefiPayloadPkg/MkFitImage.py:21]: UplVersion

Comment (gguo11837463) [reply UefiPayloadPkg/MkFitImage.py:21]: It's follow up google doc describe.

Comment (jameslu8) [reply UefiPayloadPkg/MkFitImage.py:21]: Updated to upl-version

**Review by jameslu8 — APPROVED** [review]

**Pull request merged**